src/cmd/services/m3dbnode/config/config.go

```diff
@@ -58,6 +58,16 @@ type Configuration struct {
 	Coordinator *coordinatorcfg.Configuration `yaml:"coordinator"`
 }
 
+// Validate validates the Configuration. We use this method to validate
+// fields where the validator package falls short.
+func (c Configuration) Validate() error {
+	if err := c.DB.Validate(); err != nil {
```

Comment:
Can this be simiplified to `return c.DB.Validate()`

Reply:
Yeah it could, but I did it this way intentionally so we can add a Validate method to coordinator too if need be.